Sources/FuzzilliCli/main.swift

```diff
@@ -368,6 +368,7 @@ func makeFuzzer(for profile: Profile, with configuration: Configuration) -> Fuzz
         (CodeGenMutator(),                  3),
         (InputMutator(isTypeAware: false),  2),
         (InputMutator(isTypeAware: true),   1),
+        (ConcatMutator(),                   1),
```

Comment:
The `ConcatMutator` is currently disabled (mostly) on purpose since it's just a more limited version of the `CombineMutator` (the `CombineMutator` will insert one program into another at a random position, the `ConcatMutator` does the same, but only at the end). That said, how about adding the mutator to this list with weight `0` (or commented out), with a short comment explaining that it's disabled as the `CombineMutator` is more general? Then people can easily enable it for experiments if desired.

Reply:
I have added the comment as you suggested.